modules/extensions-spring-support/src/test/resources/models/petstore.json

```diff
@@ -1053,15 +1053,15 @@
                   "name": "reconnectionStrategy",
                   "description": "A retry strategy in case of connectivity errors",
                   "modelProperties": {
+                    "org.mule.runtime.extension.internal.property.InfrastructureParameterModelProperty": {
```

Comment:
all these changes shouldn't exists. This is just falllout for the bad map type you used in the other PR. Once you fix that , most of these should be reverted